consent-ui/src/app/ais/login/login.component.ts

```diff
@@ -42,7 +42,9 @@ export class LoginComponent implements OnInit {
   }
 
   onSubmit() {
+    localStorage.setItem(ApiHeaders.COOKIE_TTL, '0');
```

Comment:
Why this line is here if we have 'CookieRenewalService'

Reply:
CookieRenewalService uses default Value as long as Value is not known. Here Value is set to 0 and if call is not answered for whatever reasyon it will stay at 0 which means, Value is not known.

Reply:
ApiHeaders.COOKIE_TTL is authId scoped
i.e.
this.sessionService.setTTL(this.authId, res.headers.get(ApiHeaders.COOKIE_TTL));
moreover localStorage is used only here